packages/tasks/src/model-libraries.ts

```diff
@@ -304,6 +304,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	"mlc-llm": {
+		prettyLabel: "MLC-LLM",
+		repoName: "MLC-LLM",
+		repoUrl: "https://github.com/mlc-ai/mlc-llm",
+		filter: true,
```

Comment:
Setting to `True` because this would effect at least 150 model checkpoints: https://huggingface.co/mlc-ai

(we'd need to make sure that all of them have `mlc-llm` tag)